src/Atmos/Model/source.jl

```diff
@@ -59,7 +59,12 @@ struct RayleighSponge{FT} <: Source
   zsponge::FT
   "Sponge Strength 0 ⩽ c_sponge ⩽ 1"
   c_sponge::FT
+  #"Reference relaxation velocity"
```

Comment:
Suggest updating the the docs  to `are relaxed to reference values (user defined) at the top boundary.` in line `53`.